core/modules/system/css/system.admin.css

```diff
@@ -90,6 +99,11 @@ span.admin-enabled {
 span.admin-missing {
   color: #f00;
 }
+@media screen and (max-width: 800px) {
+  #system-modules td.module {
+    width: 110px;
+  }
+}
```

Comment:
Ah, yeah that's misleading. I agree, we should flip those to be mobile-first as well.